pkgs/development/libraries/openldap/default.nix

```diff
@@ -16,9 +16,14 @@ stdenv.mkDerivation rec {
   buildInputs = [ openssl cyrus_sasl db groff libtool ];
 
   configureFlags =
-    [ "--enable-overlays"
+    [
       "--disable-dependency-tracking"   # speeds up one-time build
+      "--disable-ndb"
+      "--disable-perl"
+      "--disable-sql"
+      "--enable-backends=mod"
```

Comment:
Third-party modules, if any, can be loaded by absolute path (`/foo` or `/nix/store/`).